velox/vector/VariantToVector.cpp

```diff
@@ -69,7 +77,7 @@ ArrayVectorPtr variantArrayToVector(
         arrayType, variantArray, pool);
   }
 
-  return VELOX_DYNAMIC_SCALAR_TYPE_DISPATCH(
```

Comment:
`VELOX_DYNAMIC_SCALAR_TYPE_DISPATCH` now covers BIGINT and HUGEINT. We don't need the changes in this file.

Reply:
Updated, thanks!